src/app/workflow/dag/cwl-viewer/cwl-viewer.service.ts

```diff
@@ -102,18 +107,25 @@ export class CwlViewerService {
         + workflow_path);
   }
 
-  private pollJobQueue(queueUrl: string): Observable<CwlViewerDescriptor> {
-    const pollFrequencyMs = 250;
+  private pollJobQueue(queueUrl: string, onDestroy$: Subject<void>): Observable<CwlViewerDescriptor> {
+    const pollFrequencyMs = 500;
```

Comment:
Convert entire function to use pipe (because removing rxjs-compat soon), similar to https://github.com/dockstore/dockstore-ui2/blob/226c89c2f2ab7c4a3cacc3b8363ce9596490dce0/src/app/workflow/dag/cwl-viewer/cwl-viewer.service.ts, and check that it still works

Reply:
Fixed in latest commit.